ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/map.view.js

```diff
@@ -133,9 +136,50 @@ const defaultHomeBoundingBox = {
   north: 52,
 }
 
+const PanZoomViewBuilder = function(map) {
+  return Marionette.ItemView.extend({
+    template() {
+      return (
+        <React.Fragment>
+          <Keyword
+            setState={function({ polygon }) {
```

Comment:
Would this be more of an 'onChange' type of event?

Reply:
Yes but I believe state is being manipulated through `props` due to some weirdness between the two frameworks.